tests/api2/test_262_iscsi_alua.py

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env python3
```

Comment:
Is there a reason to name this test module `test_262`? Does this test module have dependencies on `test_261`?

Reply:
No reason really.  I just liked the idea of it running after 261 (which does more basic tests).

Reply:
okay, I just wanted to make sure that we aren't adding dependencies between test modules :+1: